src/foam/u2/view/SimpleSearch.js

```diff
@@ -70,9 +70,11 @@ foam.CLASS({
     function initE() {
       var self = this;
 
-      this.currentMemento_$ = this.memento.tail$;
-      if ( this.memento && ! this.memento.tail ) {
-        this.memento.tail = foam.nanos.controller.Memento.create();
+      if ( this.memento ) {
+        if ( this.memento && ! this.memento.tail ) {
```

Comment:
could you please remove `this.memento &&` from line 74? thanks!

Reply:
yeah thanks, missed that

Reply:
@olhabn addressed